src/map/atcommand.cpp

```diff
@@ -1025,21 +1025,49 @@ ACMD_FUNC(storage)
 {
 	nullpo_retr(-1, sd);
 
-	if (sd->npc_id || sd->state.vending || sd->state.buyingstore || sd->state.trading || sd->state.storage_flag)
+	if (sd->npc_id || sd->state.vending || sd->state.buyingstore || sd->state.trading)
 		return -1;
 
-	if (storage_storageopen(sd) == 1)
-	{	//Already open.
-		clif_displaymessage(fd, msg_txt(sd,250)); // You have already opened your storage. Close it first.
+	int storage = 1; // treat @storage as if it's @storage 1 [Haydrich]
```

Comment:
Wouldn't it be int32 here?
```suggestion
	int32 storage = 1; // treat @storage as if it's @storage 1 [Haydrich]
```